docs/sources/release-notes/v2.2.md

```diff
@@ -25,14 +25,27 @@ For the complete list of changes, see the [Changelog](https://github.com/grafana
 - **Configurable prefix for object storage**: Mimir can now store block data, rules, and alerts in one bucket, each under its own user-defined prefix, rather than requiring one bucket for each.
   You can configure the storage prefix by using `-<storage>.storage-prefix` option for corresponding storage: `ruler-storage`, `alertmanager-storage` or `blocks-storage`.
 
-- **Helm Chart update**: TBD
-
 - **Store-gateway can now optionally prepopulate the file system cache when memory-mapping index-header files.**
   This can help store-gateway to avoid looking stuck while loading index-headers.
   Feature can be enabled with new experimental flag `-blocks-storage.bucket-store.index-header.map-populate-enabled`.
 
 - **Faster ingester startup**: Ingesters now replay Write-Ahead-Log by about 50% faster, and they also re-join the ring sooner under some conditions.
 
+- **Helm Chart improvements**: The [Mimir Helm chart](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed) is our recommended way to install Mimir on Kubernetes. As part of the Mimir 2.2 release, we're also releasing a major version bump to the Helm chart. Below we mention some of the most notable enhancements. For the full list, see the [Helm chart changelog](FIXME LINK).
+  - The Helm chart now supports installing Mimir on [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift). 
+  - The Helm chart can now easily deploy the [Grafana Agent](https://github.com/grafana/agent) to scrape metrics and logs from all Mimir pods and ship them to a remote store, making it easy to monitor the health of your Mimir installation. For more information on how to set this up, see [FIXME](fixme - link to docs on this). 
```

Comment:
I understand that all these FIXMEs are blockers for this PR, right?

Reply:
they are and I think I replaced all of them in https://github.com/grafana/mimir/pull/2301/commits/210dca4a7fb100ccfcc21e8c2b2f17e4ae6749d6 . In https://github.com/grafana/mimir/pull/2301/commits/c68d880d5a1591fba9859039b4014eeb5335b9d4 I also removed one which talked about the smoke-test tool, which wasn't included in the getting started guide